### PR TITLE
Add missing docstrings

### PR DESCRIPTION
**Pull request by isteinbrecher**
This PR adds all missing dock strings and sets the required doctoring coverage to 100%.

@davidrudlstorfer I added temporary "TODO" dock strings to a lot of different testing functions that will likely be replaced with `pytest` fixtures in the near future.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ repos:
     hooks:
       - id: interrogate
         name: Run interrogate (linter for docstrings)
-        args: [--fail-under=95, --ignore-init-module, --style=google, -vv, meshpy/]
+        args: [--fail-under=100, --ignore-init-module, --style=google, -vv, meshpy/]
```
Comment (davidrudlstorfer) [inline .pre-commit-config.yaml:61]: If we want to also check the docstring coverage within the tests we should probably also add the path here?

See https://github.com/davidrudlstorfer/pygebege/blob/ffcb1e370bb087f2bf189bce0e376a7a77e00fdb/.pre-commit-config.yaml#L23

**Review by davidrudlstorfer — APPROVED** [review]
Very nice! Thanks for tackling this so fast! I only have a minor comment

File: .pre-commit-config.yaml (hunk 1 above)
Comment (isteinbrecher) [reply .pre-commit-config.yaml:61]: I think that is a private repository, the link does not work for me.

Comment (davidrudlstorfer) [reply .pre-commit-config.yaml:61]: Sorry my fault - https://github.com/davidrudlstorfer/pyskel/blob/a929e2b8f849fb84d19d3a7b9dbd7e575a72d446/.pre-commit-config.yaml#L48

This should work

```suggestion
        args: [--fail-under=100, --ignore-init-module, --style=google, -vv, meshpy/, tests/]
```

Basically that was my idea. I think we also need the path to the test folder if the docstring coverage should be checked?

Comment (davidrudlstorfer) [reply .pre-commit-config.yaml:61]: Sounds good - if it works out of the box let's drop the path arguments 🤝

Comment (isteinbrecher) [reply .pre-commit-config.yaml]: I tried, it works, a wrong doctoring cause the pipeline to fail. The current state should be good to go.

**Review by davidrudlstorfer — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (davidrudlstorfer) [reply .pre-commit-config.yaml]: Thanks for testing the missing docstring 🙏🏻

**Pull request merged**